### PR TITLE
chore: use changeset feedback action

### PR DESCRIPTION
**Pull request by djamaile**
Signed-off-by: djamaile <rdjamaile@gmail.com>

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by djamaile** [comment]
Will fix after today after I am done commuting!

File: .github/workflows/automate_changeset_feedback.yml [hunk 1]
```diff
@@ -26,70 +26,15 @@ jobs:
...
-                body,
-              });
-            }
+          diffRef: 'origin/master'
+          marker: <!-- changeset-feedback -->
+          issue-number: ${{ github.event.pull_request.number }}
+          botUsername: github-actions[bot]
+          app-id: ${{ secrets.BACKSTAGE_GOALIE_APPLICATION_ID }}
+          private-key: ${{ secrets.BACKSTAGE_GOALIE_PRIVATE_KEY }}
+          installation-id: ${{ secrets.BACKSTAGE_GOALIE_INSTALLATION_ID }}
```
Comment (freben) [inline .github/workflows/automate_changeset_feedback.yml:40]: This mixture of casing is a bit unfortunate. We should probably unify to all use camelCase eventually.

**Review by freben — APPROVED** [review]
Awesome, let's try it! Merging after the release though so it doesn't retrigger the Version Packages PR

File: .github/workflows/automate_changeset_feedback.yml (hunk 1 above)
Comment (Rugvip) [reply .github/workflows/automate_changeset_feedback.yml:40]: Yep although other way around, action inputs generally use `kebab-case`

Comment (djamaile) [reply .github/workflows/automate_changeset_feedback.yml:40]: Good point! I'll put on my TODO.

Comment (djamaile) [reply .github/workflows/automate_changeset_feedback.yml:40]: @freben I am also okay with parking this PR till we get the camel casing right after holidays. No high prio on this anyway. Let me know!

Comment (freben) [reply .github/workflows/automate_changeset_feedback.yml:40]: I want it in now! :)

Comment (freben) [reply .github/workflows/automate_changeset_feedback.yml:40]: Will press merge once the release is out

**Pull request merged**